src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasTest.java

```diff
@@ -172,6 +184,21 @@ protected static boolean passesMinimumThreshold(final int[] data, final int minC
     }
 
     /**
+     * Helper function to parse the genotype annotation into the SB annotation array
+     * @param arrayList the ArrayList returned from StrandBiasBySample.annotate()
+     * @return the array used by the per-sample Strand Bias annotation
+     */
+    private static int[] encodeSBBS( final List<Integer> arrayList ) {
```

Comment:
why not move this to the StrandBiasUtils class? or move all that code here? Right now it's not clear what belongs where